### PR TITLE
[DOC] update index.rst

### PR DESCRIPTION
**Pull request by Railia**
"Full debt button" explanation.

### CODE REVIEW

File: pos_debt_notebook/doc/index.rst [hunk 1]
```diff
@@ -17,8 +17,15 @@ Debt Journal
 * open POS
 * add usual products
 * select Customer
+*	set Max Debt value
```
Comment (yelizariev) [inline pos_debt_notebook/doc/index.rst]: Shall it be one space or you want to make indent?

Comment (yelizariev) [reply pos_debt_notebook/doc/index.rst]: Anyway, this feature is about to be removed. We will use Max Debt per journal

File: pos_debt_notebook/doc/index.rst [hunk 2]
```diff
@@ -17,8 +17,15 @@ Debt Journal
 * open POS
 * add usual products
 * select Customer
+*	set Max Debt value 
 * on payment screen use "Debt Journal" to pay
 
+Pay Full Debt button
+--------------------
+Pay full debt button is using when Customer has unpaid debt (Max Debt value).
+If you click on this button, then to the amount of payment for the product will add the amount of unpaid debt.
```
Comment (yelizariev) [inline pos_debt_notebook/doc/index.rst]: ``the amount of payment for the product`` what is it?

Comment (yelizariev) [reply pos_debt_notebook/doc/index.rst]: ``will add `` shall it be passive?

Comment (yelizariev) [reply pos_debt_notebook/doc/index.rst]: There are two buttons *Pay Full debt*. You need to describe both of them.

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: pos_debt_notebook/doc/index.rst (hunk 2 above)
Comment (ilmir-k) [reply pos_debt_notebook/doc/index.rst]: more correctly:
`[Pay Full Debt] ` button is displayed and may be used when Customer has unpaid debt

File: pos_debt_notebook/doc/index.rst [hunk 3]
```diff
@@ -19,6 +19,14 @@ Debt Journal
 * select Customer
 * on payment screen use "Debt Journal" to pay
 
+Pay Full Debt button
+--------------------
+The button appears when you select Customer who has unpaid debt (red amount). You see this on top of Customer's card.
+If you click this button, then you will be taken to the payment page.
+Also this button is displayed on the payment page. If you click this button, then the amount of unpaid debt will be added to the payment amount for the product.
```
Comment (yelizariev) [inline pos_debt_notebook/doc/index.rst]: Can you make it more clear, that there are two usage scenario and after being nagivating to payment page you don't need to click Button again

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: pos_debt_notebook/doc/index.rst [hunk 4]
```diff
@@ -19,6 +19,15 @@ Debt Journal
 * select Customer
 * on payment screen use "Debt Journal" to pay
 
+Pay Full Debt button
+--------------------
+The button appears when you select Customer who has unpaid debt (red amount). You see this button in 2 places: 
+ * At the top of the Customer's card.
```
Comment (yelizariev) [inline pos_debt_notebook/doc/index.rst]: Customer's form or Customer's profile, but not card

File: pos_debt_notebook/doc/index.rst [hunk 5]
```diff
@@ -19,6 +19,15 @@ Debt Journal
...
+ * On the payment page
+You can use these buttons in 2 cases:
+ * If you click on the button on the Customer's card then unpaid debt will be added to the payment amount for the product immediatly.
+ * If you don't use the button on the Customer's card  you can  use another Pay Full Debt button on the payment page.
```
Comment (yelizariev) [inline pos_debt_notebook/doc/index.rst]: It's unclear what happens when you click the button at payment page

**Review by yelizariev — CHANGES_REQUESTED** [review]
FIx format. It should not be in bold font
See https://github.com/it-projects-llc/pos-addons/blob/8636cdf9307cb9922b51cd47b3c0e4c063938436/pos_debt_notebook/doc/index.rst

And https://odoo-development.readthedocs.io/en/latest/dev/docs/rst-requirements.html#extra-lines

File: pos_debt_notebook/doc/index.rst [hunk 6]
```diff
@@ -19,6 +19,15 @@ Debt Journal
 * select Customer
 * on payment screen use "Debt Journal" to pay
 
+Pay Full Debt button
+--------------------
+The button appears when you select Customer who has unpaid debt (red amount). You see this button in 2 places: 
+ * At the top of the Customer's card.
+ * On the payment page
+You can use these buttons in 2 cases:
+ * If you click on the button on the Customer's card then unpaid debt will be added to the payment amount for the product immediatly.
```
Comment (ilmir-k) [inline pos_debt_notebook/doc/index.rst]: immediately

**Comment by ilmir-k** [comment]
@Railia please fix all @yelizariev's requests
 

> https://odoo-development.readthedocs.io/en/latest/dev/docs/rst-requirements.html#extra-lines
>  Customer's form

File: pos_debt_notebook/doc/index.rst [hunk 7]
```diff
@@ -19,6 +19,19 @@ Debt Journal
...
+
+The button appears when you select Customer who has unpaid debt (red amount). You see this button in 2 places: 
+
+* At the top of the Customer's form.
+* On the payment page
+ 
+You can use these buttons in 2 cases:
+
+* If you click on the button on the Customer's form then unpaid debt will be added to the payment amount for the product immediately.
+* If you don't use the button on the Customer's form  you can  use another ``[Pay Full Debt]`` button on the payment page. If you click on the button then unpaid debt will be added to the payment amount for the product.
```
Comment (yelizariev) [inline pos_debt_notebook/doc/index.rst]: I don't understand what are difference between actions after clicking any of those buttons. If there are  no difference, we probably shall not repeat about two different places again

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Review by yelizariev — APPROVED** [review]
@ilmir-k check it too please

**Review by ilmir-k — APPROVED** [review]

**Comment by ilmir-k** [comment]
@yelizariev it can be merged

File: pos_debt_notebook/doc/index.rst [hunk 8]
```diff
@@ -24,10 +24,10 @@ Pay Full Debt button
 
 The button appears when you select Customer which has unpaid debt (red amount). You can see this button in 2 places: 
 
-* At the top of the Customer's form.
+* At the top of the Customer's form
 * On the payment page
  
-When you click ``[Pay Full Debt]`` button, debt amount will be added to the payment list with negative sign. That negative amount has to be covered by normal payment (e.g. by cash). After that the Customer will have zero debt value.
+When you click ``[Pay Full Debt]`` button, the debt amount will be added to the payment list with negative sign. This negative amount has to be covered by normal payment (e.g. by cash). After that the Customer will have zero debt value.
```
Comment (yelizariev) [inline pos_debt_notebook/doc/index.rst]: Actually, it's ``That`` https://english.stackexchange.com/questions/42306/this-vs-that

**Pull request merged**